src/components/user/login/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import {
+  Grid, Row, Col, Alert,
+} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+import { loginAPI } from '../../../store/actions/login';
+import { ROUTES } from '../../../constants';
+import LoginForm from './login-form';
+
+class Login extends Component {
+  constructor() {
+    super();
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(fields) {
+    const { login } = this.props;
+    login(fields);
+  }
+
+  renderFormError() {
+    const { loginState } = this.props;
+    const { message } = loginState;
+
+    return message ? <Alert bsStyle="danger">{message}</Alert> : null;
+  }
+
+  render() {
+    const { loginState } = this.props;
+    const { statusCode, data } = loginState;
+    const { home } = ROUTES;
+    const isLoginSuccessful = statusCode === 200;
+    const loginFormProps = {
+      title: 'Login',
+      onSubmit: this.handleSubmit,
+      inputFields: [
+        { textInput: { name: 'email' } },
+        { textInput: { name: 'password' } },
+      ],
+    };
+
+    if (isLoginSuccessful) {
+      localStorage.setItem('token', data);
```

Comment:
Any reason why you went with localStorage vs cookie vs session storage?

From our previous discussion, it is important to understand the pros, cons, limitations etc of each of these storage methods and ensure that you're using the correct one that achieves what you want.

Reply:
local storage vs session storage, I think this one might be kinda self-explanatory but I just wanted to have to token be more persistent. This is just personal preference. As a user, if I am logged in to a page and I accidentally close that tab, when I open a new tab and navigate to the same page I would like to stay logged in. 

local storage vs cookie, while I think cookies are a better option, I wanted to keep the implementation more simple and local storage just achieve what I want for now. Once I look to expand my application, I will re-consider using cookies.

Reply:
👍